content/advanced/fine-grained-reactivity.mdx

```diff
@@ -0,0 +1,248 @@
+# Understanding Fine-Grained Reactivity
+
+Reactivity in programming establishes automatic responses to data changes, eliminating the need for manual updates to the user interface (UI). 
+By connecting UI elements to the underlying data, updates become automated.
+
+The term "fine-grained reactivity" refers to systems that allow for highly targeted behavioral changes, in contrast to approaches that only support broader, and sometimes unnecessary, updates.
```

Comment:
Maybe worth mentioning an example like "if an attribute has to be updated, only that attribute is updated, directly, with Solid, while for example the entire component is re-executed first with React".